packages/controllers/src/snaps/SnapController.ts

```diff
@@ -1718,7 +1719,7 @@ export class SnapController extends BaseController<
     }
   }
 
-  private async _startSnap(snapData: SnapData) {
+  private async _startSnap(snapData: { snapId: string; sourceCode: string }) {
```

Comment:
Why did you remove this type?

Reply:
Because its name was vague, collided with completely different `SnapData` type in execution environments, and (most importantly) wasn't used anywhere but here.